src/main/java/org/folio/inventory/support/http/server/JsonResponse.java

```diff
@@ -67,6 +71,12 @@ public static void unprocessableEntity(HttpServerResponse response, String error
     response(response, new JsonObject().put("errors", error), 422);
   }
 
+  public static void someOfEntitiesNotFound(HttpServerResponse response, List<String> ids) {
+    JsonObject errors = new JsonObject();
+    errors.put("ids", ids);
+    response(response, new JsonObject().put("errors", errors), 404);
+  }
```

Comment:
If this is using the standard JSON error representation, this might be better communicated to the client by using the 422 status code instead.